Build/Customizations/Projects.props

```diff
@@ -2,9 +2,49 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <TestingSetupBuildFile>$(SolutionDirectory)Build\TestingSetupForDatabase.build</TestingSetupBuildFile>
+    <TestSetupBuildFileForDatabase>$(SolutionDirectory)Build\TestSetupForDatabase.build</TestSetupBuildFileForDatabase>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <DatabaseTestConfiguration Condition="'$(DatabaseTestConfiguration)' == ''">
+      Win_NET48 + NET48 + NoBrowser + SqlServer2012 + Debug + x86;
+      Win_NET48 + NET48 + NoBrowser + SqlServer2012 + Release + x86;
+      Win_NET48 + NET48 + NoBrowser + SqlServer2012 + Debug + x64;
+      Win_NET48 + NET48 + NoBrowser + SqlServer2012 + Release + x64;
+
+      <!-- Local-->
+      LocalMachine + NET48 + NoBrowser + SqlServerDefault + Debug + x86;
+      LocalMachine + NET48 + NoBrowser + SqlServerDefault + Release + x86;
+      LocalMachine + NET48 + NoBrowser + SqlServerDefault + Debug + x64;
+      LocalMachine + NET48 + NoBrowser + SqlServerDefault + Release + x64;
+
+      <!-- Exercise compatibility between installed .NET version, target framework and SQL Server -->
+      Win_NET48 + NET472 + NoBrowser + SqlServer2014 + Release + x64;<!-- Change to SqlServer2019 once it exists -->
+      Win_NET48 + NET462 + NoBrowser + SqlServer2017 + Release + x64;
+      Win_NET472 + NET472 + NoBrowser + SqlServer2016 + Release + x64;
+      Win_NET472 + NET462 + NoBrowser + SqlServer2014 + Release + x64;
```

Comment:
the order is different between database and normal but there's no obvious reason for this.

Reply:
The tests for SQL2005, SQL2008, SQL2008R2 are missing.

     Win_NET48 + NET472 + NoBrowser + SqlServer2014 + Release + x64;<!-- Change to SqlServer2019 once it exists -->
      Win_NET48 + NET462 + NoBrowser + SqlServer2017 + Release + x64;
      Win_NET472 + NET472 + NoBrowser + SqlServer2016 + Release + x64;
      Win_NET472 + NET462 + NoBrowser + SqlServer2014 + Release + x64;
      Win_NET462 + NET462 + NoBrowser + SqlServer2008R2 + Release + x64;
      Win_NET462 + NET462 + NoBrowser + SqlServer2008 + Release + x64;
      Win_NET462 + NET462 + NoBrowser + SqlServer2005 + Release + x64;

Reply:
Fun fact: we now have 11 database configs vs the original 28 configs. this should save us about 7 minutes of build time.